distro/src/lib.rs

```diff
@@ -0,0 +1,32 @@
+pub const TRUST_GRAPH_WASM: &'static [u8] = include_bytes!("../registry-service/registry.wasm");
+pub const SQLITE_WASM: &'static [u8] = include_bytes!("../registry-service/sqlite3.wasm");
+pub const CONFIG: &'static [u8] = include_bytes!("../registry-service/Config.toml");
+pub const SCHEDULED_SCRIPT_1: &'static str = include_str!("../registry-service/air/registry-scheduled-scripts.clearExpired_86400.air");
+pub const SCHEDULED_SCRIPT_1_PERIOD: u32 = 86400;
+
+pub const SCHEDULED_SCRIPT_2: &'static str = include_str!("../registry-service/air/registry-scheduled-scripts.renew_43200.air");
+pub const SCHEDULED_SCRIPT_2_PERIOD: u32 = 43200;
+
+pub const SCHEDULED_SCRIPT_3: &'static str = include_str!("../registry-service/air/registry-scheduled-scripts.replicate_3600.air");
+pub const SCHEDULED_SCRIPT_3_PERIOD: u32 = 3600;
```

Comment:
I think names could be better. `CLEAR_EXPIRED_AIR`, `RENEW_AIR`, `REPLICATE_AIR`. Same with periods. And please specify "sec", ie `period_sec`